### DISCUSSION TITLE
Is there a list of supported TLS Ciphers for ETCD?

### DISCUSSION THREAD

**Discussion by NBK27** [etcd-io/etcd#16402] [Q&A] [answered]
Is there a list of supported TLS Ciphers for ETCD?

**Comment by jmhbnz** [comment, marked as answer]
Hey @NBK27 - Thanks for your question. etcd currently relies on the list of cipher suites provided by golang standard `crypto/tls` library, please refer to:
- https://cs.opensource.google/go/go/+/refs/tags/go1.20.7:src/crypto/tls/cipher_suites.go;l=53
- https://cs.opensource.google/go/go/+/refs/tags/go1.20.7:src/crypto/tls/cipher_suites.go;l=82

https://github.com/etcd-io/etcd/blob/1edd8baf3c6024eceb8b0d45062f0cebf7be9bc5/server/embed/config.go#L226-L229
